product-tours/tour-tf-keras/Tour-With-TensorFlow-and-Keras.ipynb

```diff
@@ -0,0 +1,2217 @@
+   "source": [
+    "! pip install neptune-client==0.4.128 neptune-contrib==0.24.9 tensorflow==2.3.1 scikit-plot==0.3.7 matplotlib==3.3.3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "DO7yawFVxVIt",
+    "tags": [
+     "installation",
+     "library_updates"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! pip install neptune-client neptune-contrib tensorflow scikit-plot matplotlib --upgrade"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 1: Import Libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import hashlib\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import neptune\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import tensorflow as tf\n",
+    "from neptunecontrib.api import log_table\n",
+    "from neptunecontrib.monitoring.keras import NeptuneMonitor\n",
+    "from scikitplot.metrics import plot_roc, plot_precision_recall"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 2: Select project"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "neptune.init('shared/tour-with-tf-keras-tests',\n",
+    "             api_token='ANONYMOUS')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "rf3FmHL11R0r",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 3: Prepare params"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "ljhFSnypxdUW",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "parameters = {'dense_units': 32,\n",
+    "              'activation': 'relu',\n",
+    "              'dropout': 0.3,\n",
+    "              'learning_rate': 0.05,\n",
+    "              'batch_size': 32,\n",
+    "              'n_epochs': 10}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "AW4MM4qj5ErX",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 4: Create experiment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 71
+    },
+    "colab_type": "code",
+    "id": "GvDSBSrOx-R4",
+    "outputId": "667d1b2f-cdef-44f0-f505-832205dea223",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "neptune.create_experiment(name='keras-training',\n",
```

Comment:
I think this would fail when running (main.py isn't there). 
Also in the basic version I'd only have params and name maybe.

Reply:
changed.